webboot/webboot.go

```diff
@@ -32,10 +32,11 @@ var (
 	verbose  = flag.Bool("verbose", false, "Verbose output")
 	ipv4     = flag.Bool("ipv4", true, "use IPV4")
 	ipv6     = flag.Bool("ipv6", true, "use IPV6")
+	dryrun   = flag.Bool("dryrun", false, "Do not do the kexec")
 	bookmark = map[string]*webboot.Distro{
-		"tinycore": &webboot.Distro{"/boot/vmlinuz64", "/boot/corepure64.gz", "console=tty1", "http://tinycorelinux.net/10.x/x86_64/release/CorePure64-10.1.iso"},
+		"tinycore": &webboot.Distro{"boot/vmlinuz64", "/boot/corepure64.gz", "console=tty0", "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-current.iso"},
+		"mykernel": &webboot.Distro{"/bzImage", "/boot/corepure64.gz", "console=tty0", "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-current.iso"},
```

Comment:
name this something more meaningful than "mykernel" :)

Reply:
You have to talk to Ron about that.

Reply:
"mykernel" clearly tells you that the program is booting using the users kernel and not the one packaged originally into the iso.Can you suggest some other names which might work here?